File: customer/ShippingStarted.py

from AbstractEvent import AbstractEvent
import json

class ShippingStarted(AbstractEvent):
    id : int
    productName : str
    deliveryAddress : str
    orderId : str
    phoneNumber : int
    deliveryDate : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.productName = None
        self.deliveryAddress = None
        self.orderId = None
        self.phoneNumber = None
        self.deliveryDate = None

File: app/src/main/java/shoesdelivery/domain/ShippingStarted.java
package shoesdelivery.domain;

import shoesdelivery.domain.*;
import shoesdelivery.infra.AbstractEvent;
import lombok.*;
import java.util.*;
@Data
@ToString
public class ShippingStarted extends AbstractEvent {

    private Long id;
    private String productName;
    private String deliveryAddress;
    private String orderId;
    private Long phoneNumber;
    private String deliveryDate;
}


